quodlibet/quodlibet/ext/events/randomalbum.py

```diff
@@ -190,10 +190,17 @@ def plugin_on_song_started(self, song):
                 nr_albums = min(total, max(int(0.03 * total), 3))
                 print_d("Choosing from %d library albums. Best:" % nr_albums)
                 chosen_albums = random.sample(values, nr_albums)
-                album_scores = sorted(self._score(chosen_albums))
+                album_scores = self._score(chosen_albums)
                 for score, album in album_scores[-5:]:
                     print_d("%0.1f scored by %s" % (score, album("album")))
```

Comment:
This print should probably be removed, since without the sort it just prints five random albums as opposed to the highest rated ones (the print above would also need to be modified a little, removing the 'Best'). Perhaps print the chosen album instead.